dbt/models/iasworld/schema/iasworld.pardat.yml

```diff
@@ -259,6 +259,14 @@ sources:
             description: '{{ doc("shared_column_year") }}'
           - name: tieback
             description: '{{ doc("shared_column_tieback_key_pin") }}'
+            tests:
+              - expression_is_true: 
+                  name: iasworld_pardat_tieback_does_not_contain_hyphen
+                  expression: |
+                    not like '%-%'
```

Comment:
**[Nitpick, non-blocking]** The pipe character in this context indicates a multiline string, but since the string itself is short enough to fit on one line, there shouldn't be a need for it:

```suggestion
                  expression: not like '%-%'
```